lms/templates/discussion/_discussion_inline.html

```diff
@@ -12,3 +12,15 @@
         <button class="new-post-btn btn-neutral btn-small">${_("Add a Post")}</button>
     % endif
 </div>
+<script type="text/javascript">
+/* global DiscussionModuleView */
```

Comment:
Should this be a DiscussionXBlockView?  Does it deal with tabs as well, or is this for in-line xblock?